packages/@aws-cdk/aws-eks/test/integ.eks-cluster.defaults.expected.json

```diff
@@ -1229,7 +1229,7 @@
               },
               "/",
               {
-                "Ref": "AssetParameters4dd58c778ce9bf0be6031537b4555ac9cbdd628b5770b5ec68016ff16b44428eS3Bucket3E4D83F8"
```

Comment:
Why were these changed!? You haven’t updated the custom resource code?

Reply:
Yes I have bumped the layer version from 2.0.0-beta1 to 2.0.0-beta2 in this PR so all  regions of classic partition as well as `aws-cn` will be using `2.0.0-beta2`.
https://github.com/aws-samples/aws-lambda-layer-kubectl/releases/tag/2.0.0-beta2

Is it the reason the references were changed? Should I do this in another PR?

https://github.com/aws/aws-cdk/blob/d202e796be8e309234e171d89aec8ec0ef294546/packages/%40aws-cdk/aws-eks/lib/kubectl-provider.ts#L37

Reply:
OK got it. Yes, the template of the nested stack changed and therefore its artifact ID was invalidated.

I am okay with doing this in this PR. But... when you update your branch you'll notice that I've consolidated all integ tests into one, so your life is going to be easier (#6636).